From 542bb52a97598113224171b28c34813403a2fadf Mon Sep 17 00:00:00 2001
From: Roman Shaposhnik <rvs@zededa.com>
Date: Sat, 24 Aug 2019 19:22:49 -0700
Subject: [PATCH 2/3] Fixing small bugs

Signed-off-by: Roman Shaposhnik <rvs@zededa.com>
---
 .mvn/wrapper/MavenWrapperDownloader.java | 4 ++--
 mvnw                                     | 6 +++---
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/.mvn/wrapper/MavenWrapperDownloader.java b/.mvn/wrapper/MavenWrapperDownloader.java
index 69e2ece3e..a2fae998b 100644
--- a/.mvn/wrapper/MavenWrapperDownloader.java
+++ b/.mvn/wrapper/MavenWrapperDownloader.java
@@ -25,7 +25,7 @@ public class MavenWrapperDownloader {
      * Default URL to download the maven-wrapper.jar from, if no 'downloadUrl' is provided.
      */
     private static final String DEFAULT_DOWNLOAD_URL = "https://repo.maven.apache.org/maven2/io/takari/maven-wrapper/"
-        + WRAPPER_VERSION + "/maven-wrapper-" + WRAPPER_VERSION + " .jar";
+        + WRAPPER_VERSION + "/maven-wrapper-" + WRAPPER_VERSION + ".jar";
 
     /**
      * Path to the maven-wrapper.properties file, which might contain a downloadUrl property to
@@ -114,4 +114,4 @@ public class MavenWrapperDownloader {
         rbc.close();
     }
 
-}
\ No newline at end of file
+}
diff --git a/mvnw b/mvnw
index 197b60827..afe2c11c0 100755
--- a/mvnw
+++ b/mvnw
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # ----------------------------------------------------------------------------
 # Licensed to the Apache Software Foundation (ASF) under one
 # or more contributor license agreements.  See the NOTICE file
@@ -211,7 +211,7 @@ else
     if [ "$MVNW_VERBOSE" = true ]; then
       echo "Couldn't find .mvn/wrapper/maven-wrapper.jar, downloading it ..."
     fi
-    if [ "$MVNW_REPOURL" = true]; then
+    if [ "$MVNW_REPOURL" = true ]; then
       jarUrl="$MVNW_REPOURL/io/takari/maven-wrapper/0.5.2/maven-wrapper-0.5.2.jar"
     else
       jarUrl="https://repo.maven.apache.org/maven2/io/takari/maven-wrapper/0.5.2/maven-wrapper-0.5.2.jar"
@@ -302,4 +302,4 @@ exec "$JAVACMD" \
   $MAVEN_OPTS \
   -classpath "$MAVEN_PROJECTBASEDIR/.mvn/wrapper/maven-wrapper.jar" \
   "-Dmaven.home=${M2_HOME}" "-Dmaven.multiModuleProjectDirectory=${MAVEN_PROJECTBASEDIR}" \
-  ${WRAPPER_LAUNCHER} $MAVEN_CONFIG "$@"
\ No newline at end of file
+  ${WRAPPER_LAUNCHER} $MAVEN_CONFIG "$@"
-- 
2.15.1

